lib/modules/datasource/packagist/schema.ts

```diff
@@ -45,18 +40,13 @@ export const MinifiedArray = z.array(z.record(z.unknown())).transform((xs) => {
 });
 export type MinifiedArray = z.infer<typeof MinifiedArray>;
 
-export const ComposerRelease = z
-  .object({
-    version: z.string(),
-  })
-  .merge(
-    looseObject({
-      homepage: z.string(),
-      source: z.object({ url: z.string() }),
-      time: z.string(),
-      require: z.object({ php: z.string() }),
-    })
-  );
+export const ComposerRelease = z.object({
+  version: z.string(),
+  homepage: z.optional(looseValue(z.string())),
```

Comment:
🤔 isn't loose value already optional?

Reply:
No, I've made those helpers to fallback for nulls only for consistency, without using undefined

Reply:
Well, I see, I think I have to adjust tests instead of using optional